### ISSUE TITLE
API error

### ISSUE THREAD

**Issue by james7313** [DevMiser/Lumina#3]
Hello,  I have followed your instructions with an RPI4B+ 8GB. Lumina starts just fine, I think, after everything installs without any errors. It detects me when I say Lumina and it flashes what I say up on the screen, then it says API error, try again later. I am attaching a video for you to see.  I have confirmed the openai api is correct and regenerated it, neither works.  The picovoice secure key seems to be working fine.  Any suggestions?

https://github.com/DevMiser/Lumina/assets/23411438/ded7b44e-cec7-4a0f-b19f-758774d833e1

**Comment by DevMiser** [comment]
Hi James.  I agree that the fact that Lumina is detecting its wake word and is displaying what you asked it to generate means that there is no issue with the Picovoice side.  The API error message is on the OpenAI side.  This error message is associated with OpenAi's servers being too busy, the API key not being accepted, or the rate limit being exceeded.  One issue may be that your API key is not associated with a payment method.  Please double check that.

**Comment by james7313** [comment]
OK, thanks for your quick response. My payment method has expired so I
updated it and will give it time. I think it is a function of the server so
I will try throughout the next few days.

James

**Comment by derfeznoj** [comment]
I have similar problems. The OpenAI billing system won't accept any of my credit cards to update billing. I've found dozens of posts on the dev forums with people having the same problems. They say it can take several weeks for help to get back to you. Some folks say it's because the billing zip is too far away from the IP location. Go figure. Any thoughts about implementing the StableDiffusion model that runs native on RaspPI?

**Comment by james7313** [comment]
Yeah, I'm going to submit a ticket today, I'm still getting the same thing
and it is definitely on the openai side...

**Comment by derfeznoj** [comment]
I was able to add a billing charge record this morning, maybe it's working again.

**Comment by harrisbrodie** [comment]
I contacted OpenAI support to allow me to convert my account to paid from free and was then able to add my billing info. Lumina is now working with the OpenAI API.

**Comment by DevMiser** [comment]
Great to hear.  I hope you enjoy it.

**Comment by james7313** [comment]
Yeah, I had to regenerate the openai API after updating my account to paid.
Is this saving the images anywhere or is there a way to have it save them
locally?

**Comment by DevMiser** [comment]
I will add the ability to save the images in the near future.
